sanic_openapi/__init__.py

```diff
@@ -1,4 +1,2 @@
-from .swagger import swagger_blueprint
-
 __version__ = "0.6.0"
-__all__ = ["swagger_blueprint"]
+__all__ = []
```

Comment:
I think we don't need `__all__` variable anymore if we remove the `swagger_blueprint` from it.
BTW, to make it more intuitive,  how about let users import `Swagger()` directly?
Like this:
```python
from sanic_openapi import Swagger
```

Reply:
Done